l10n_br_sale_product/models/sale.py

```diff
@@ -134,65 +133,91 @@ def _set_amount_costs(self):
                     line.price_gross,
                     self.amount_costs,
                     line.order_id.amount_gross),
-                })
+            })
         return True
 
-    ind_pres = fields.Selection([
-        ('0', u'Não se aplica (por exemplo,'
-              u' Nota Fiscal complementar ou de ajuste)'),
-        ('1', u'Operação presencial'),
-        ('2', u'Operação não presencial, pela Internet'),
-        ('3', u'Operação não presencial, Teleatendimento'),
-        ('4', u'NFC-e em operação com entrega em domicílio'),
-        ('5', u'Operação presencial, fora do estabelecimento'),
-        ('9', u'Operação não presencial, outros')], u'Tipo de operação',
-        readonly=True, states={'draft': [('readonly', False)]},
+    ind_pres = fields.Selection(
+        selection=[('0', u'Não se aplica (por exemplo,'
+                         u' Nota Fiscal complementar ou de ajuste)'),
+                   ('1', u'Operação presencial'),
+                   ('2', u'Operação não presencial, pela Internet'),
+                   ('3', u'Operação não presencial, Teleatendimento'),
+                   ('4', u'NFC-e em operação com entrega em domicílio'),
+                   ('5', u'Operação presencial, fora do estabelecimento'),
+                   ('9', u'Operação não presencial, outros')],
+        string=u'Tipo de operação',
+        readonly=True,
+        states={'draft': [('readonly', False)]},
         required=False,
         help=u'Indicador de presença do comprador no estabelecimento \
-             comercial no momento da operação.', default=_default_ind_pres)
+             comercial no momento da operação.',
+        default=_default_ind_pres)
     amount_untaxed = fields.Float(
-        compute='_amount_all_wrapper', string='Untaxed Amount',
-        digits=dp.get_precision('Account'), store=True,
-        help="The amount without tax.", track_visibility='always')
+        compute='_amount_all_wrapper',
+        string='Untaxed Amount',
+        digits=dp.get_precision('Account'),
+        store=True,
+        help="The amount without tax.",
+        track_visibility='always')
     amount_tax = fields.Float(
-        compute='_amount_all_wrapper', string='Taxes', store=True,
-        digits=dp.get_precision('Account'), help="The tax amount.")
+        compute='_amount_all_wrapper',
```

Comment:
back to v7 or v8, because of the old API, that _amount_all_wrapper wrapper was needed as you can see here:
https://github.com/odoo/odoo/blob/8.0/addons/sale/sale.py#L59
However in v10, with the genaralized API this is not required anymore as you can see here:
https://github.com/odoo/odoo/blob/10.0/addons/sale/models/sale.py#L22 (no more _amount_all_wrapper method)

So you could please delete this _amount_all_wrapper and use _amount_all directly?
Also at the beginning of the _amount_all metho override, please add the following comment:

"This override is specific for Brazil. Idealy, starting from v12 we should just call super when the sale is not for Brazil".

(This multi-localization ability is very much required. However, Akretion won't spend efforts in doing them for v10).